transport/grpc/headers.go

```diff
@@ -69,9 +69,9 @@ const (
 	// _applicationErrorNameHeader is the header for the name of the application
 	// error.
 	_applicationErrorNameHeader = "rpc-application-error-name"
-	// _applicationErrorNameHeader is the header for the the application error
-	// message.
-	_applicationErrorMessageHeader = "rpc-application-error-message"
+	// _applicationErrorDetailsHeader is the header for the the application error
+	// meta details string.
+	_applicationErrorDetailsHeader = "rpc-application-error-details"
```

Comment:
What is the implication of this change? Client might be impacted by this :o

Reply:
Whoops, good catch. I've updated the summary:
```
Note that this is not a breaking change since this feature is
unreleased.
```